app/2.3.x/external-plugins.md

```diff
@@ -0,0 +1,248 @@
+---
+title: External Plugins Support
```

Comment:
I believe the "External" nomenclature used here doesn't help understanding what they are. I would suggest calling this "Plugins in other languages". That's what most people care about, and that's what they will Google for when trying to find information about this. The fact that they need to be implemented on an external process is an implementation detail, not the main thing.

Reply:
agree